### ISSUE TITLE
Unable to Escape characters

### ISSUE THREAD

**Issue by evenflow58** [acuminous/yadda#160]
When you are looking at a HTML that has multiple lines webdriveIO gets the text 
blah blah blah 
blah blah blah
as 
blah blah blah \nblah blah blah

Unfortunately "blah blah blah \nblah blah blah" is parsed as "blah blah blah \\nblah blah blah" so there does not seem to be any way to match them.

**Comment by cressie176** [comment]
I'm working on multiline steps at the moment. It's not quite ready yet, however the version of yadda in origin/master supports this. See [here](https://github.com/acuminous/yadda/blob/master/examples/multiline-examples/features/poem.feature) for an example.

However I should stress that I'm still not entirely satisfied with the the implementation. The tests output from mocha isn't as friendly as I would like and it's not nice having to define your steps with a \n

```js
.define("Good Times\n$poem", function (poem, next) {
  // Code goes here
})
```
so some things might change before I release it. If you would prefer to wait for the npm release then your best options are...

1. Put the expected output in a file and reference the file from the step
2. Write a step which collects the individual lines, and a final step which verifies the actual webdriver output with the collected lines

**Comment by cressie176** [comment]
OK to close?

**Comment by cressie176** [comment]
I've released yadda 0.15.2 which includes support for multiline steps.

**Comment by evenflow58** [comment]
Awesome. Thanks!

Sent from my iPhone

On Jul 29, 2015, at 7:53 AM, Stephen Cresswell <notifications@github.com>
wrote:

I've released yadda 0.15.2 which includes support for multiline steps.

—
Reply to this email directly or view it on GitHub
<https://github.com/acuminous/yadda/issues/160#issuecomment-125929441>.
